source/apiVolontaria/apiVolontaria/tests/tests_view_Users.py

```diff
@@ -537,6 +536,13 @@ def test_list_users(self):
 
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_view_users_with_filter(self):
```

Comment:
Plese use more accurate name or follow the actual standard so it's easy to maintain.
Here you want to `list` users not `view` and you use a specific filter

`test_list_users_filtered_by_search`

Reply:
Yes okay